src/ohri-form.component.tsx

```diff
@@ -169,6 +164,30 @@ const OHRIForm: React.FC<OHRIFormProps> = ({
     reportError(patientError, t);
   }, [patientError, t]);
 
+  const WarningModal = () => {
+    return (
+      <ComposedModal preventCloseOnClickOutside open={true} onClose={() => setShowWarningModal(false)}>
```

Comment:
This modal gets shown when the user enters data into a form but then tries to navigate away from the form without first saving it.